lib/ui/home/home_screen.dart

```diff
@@ -159,7 +164,7 @@ class HomeScreenState extends ConsumerState<HomeScreen> {
   @override
   void initState() {
     super.initState();
-    ref.read(homeViewModelProvider.notifier).getSurveys(isRefresh: true);
+    ref.read(homeViewModelProvider.notifier).getSurveys();
```

Comment:
According to [the requirement](https://www.notion.so/nimblehq/Internal-Certification-Path-Mobile-de354e1ce76f4b80922c3db21ea0117f?pvs=4#6c17b6459ffe4df28a06b917d37d0277):

> The cached data will be used to display on the next time users enter the Home screen, or relaunch; in the meantime, the app should fetch data from the API as well, if there are any changes, then data should be propagated to the Home data.

I think we should still need to fetch the survey list here. However, we can opt out the requirement a bit to keep the app simple:

> ~if there are any changes, then data should be propagated to the Home data~ After fetching completed, then data should be propagated to the Home data.

Which means no need to compare the changes. What do you think?

Reply:
Nice catch. I prefer fetch everytime, if failed to fetch then load from cache